signac/import_export.py

```diff
@@ -1144,7 +1145,12 @@ def read_statepoint_file(path):
             "The jobs identified with the given schema function are not unique!"
         )
 
-    tarfile.extractall(path=tmpdir)
+    if sys.version_info[1] >= 12:  # minor version
```

Comment:
Compare the major.minor as a tuple (I tested that this snippet should work).
```suggestion
    if sys.version_info[:2] >= (3, 12):
```